### PR TITLE
review wanted

### PR DESCRIPTION
**Pull request by kajsabootin**
Link to my deployed project:
https://kajsassurvey.netlify.com/

My reflections on how this week's project turned out:
I am most satisfied that I got all the requirements to work.

Things I'd like to have clarified or explained in more detail:
I would like to have that you not can submit before you have pick an answer on all questions. I would like to have feedback on how to best solve that.

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,81 @@
...
+
+        <form onSubmit={event => event.preventDefault()}>
+
+          <section className="firstSection">
+
+          <p>Which type of pizza you prefer?</p>
+          <select className="formDropdown"
+            onChange={event => setPizza(event.target.value)}
+            value={pizza}
+          >
```
Comment (ericaglimsholt) [inline code/src/App.js]: Will make it easier to read
```suggestion
          <select 
            className="formDropdown"
            onChange={event => setPizza(event.target.value)}
            value={pizza}
          >
```

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,81 @@
...
+            <option value="">Choose your favorite!</option>
```
Comment (ericaglimsholt) [inline code/src/App.js]: I guess this isn't a option
```suggestion
            <option value="" disabled >Choose your favorite!</option>
```

File: code/src/App.js [hunk 3]
```diff
@@ -1,9 +1,81 @@
...
+            <option value="Margarita">Margarita</option>
+            <option value="Calzone">Calzone</option>
+            <option value="Vesuvio">Vesuvio</option>
+          </select>
+
+          </section>
+
+          <section className="secondSection">
+
+          <p>Which sauce do you prefer for your pizza?</p>
```
Comment (ericaglimsholt) [inline code/src/App.js]: Would suit good as a title!

File: code/src/App.js [hunk 4]
```diff
@@ -1,9 +1,81 @@
...
+                {favorite}
+              </label>
+            ))}
+          </div>
+
+          <section className="buttonSection">
+
+          <button className="submitButton"
+            onClick={ () => setSubmitted(true)}
+          >SUBMIT
```
Comment (ericaglimsholt) [inline code/src/App.js]:
```suggestion
          <button 
            className="submitButton"
            onClick={ () => setSubmitted(true)}
          >SUBMIT
```

File: code/src/Header.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
```
Comment (ericaglimsholt) [inline code/src/Header.js]: Empty component?

File: code/src/Summary.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import "./index.css";
+
+export const Summary = (props) => {
+
+    return (
+
+        <section className="Summary">
```
Comment (ericaglimsholt) [inline code/src/Summary.js]:
```suggestion
        <section className="summary">
```

File: code/src/Header.js (hunk 5 above)
Comment (kajsabootin) [reply code/src/Header.js]: I have put my image in the header, so if I remove that component. Will the picture not disappear then?

**Pull request closed without merging**